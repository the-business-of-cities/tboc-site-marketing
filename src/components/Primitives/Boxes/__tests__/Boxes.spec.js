import { Boxes } from "../index";
import React from "react";
import { shallow } from 'enzyme';

describe("Boxes", () => {
  it("renders correctly", () => {
    expect(shallow(<Boxes />)).toMatchSnapshot();
  });
});
