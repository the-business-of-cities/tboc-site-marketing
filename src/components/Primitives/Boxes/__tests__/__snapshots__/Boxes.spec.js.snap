// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Boxes renders correctly 1`] = `
<styled.div>
  <Box
    image={
      Object {
        "url": "https://images.pexels.com/photos/220444/pexels-photo-220444.jpeg?h=350&dpr=2&auto=compress&cs=tinysrgb",
      }
    }
    key="0"
    link="#"
    text="Watch things"
    title="Watch"
  />
  <Box
    image={
      Object {
        "url": "https://images.pexels.com/photos/220444/pexels-photo-220444.jpeg?h=350&dpr=2&auto=compress&cs=tinysrgb",
      }
    }
    key="1"
    link="#"
    text="Read things"
    title="Read"
  />
  <Box
    image={
      Object {
        "url": "https://images.pexels.com/photos/220444/pexels-photo-220444.jpeg?h=350&dpr=2&auto=compress&cs=tinysrgb",
      }
    }
    key="2"
    link="#"
    text="Listen to things"
    title="Listen"
  />
</styled.div>
`;
